src/test/kotlin/gwent/vanilla/AppTest.kt

```diff
@@ -0,0 +1,11 @@
+package gwent.vanilla
+
+import kotlin.test.Test
+import kotlin.test.assertTrue
+
+class AppTest {
+    @Test
+    fun dummyTest() {
+        assertTrue { true }
+    }
+}
```

Comment:
Useless test

Reply:
Yes, but just placeholder and useful for testing if JUnit and Gradle works - I vote it stays for the next five commits until proper tests are implemented.